pages/company/company.pixy

```diff
@@ -13,6 +13,30 @@ component CompanyPage(company *arn.Company, studioAnime, producedAnime, licensed
 			CompanyAnimes("Licensed:", licensedAnime, user)
 		
 		.company-sidebar
+			h3.mountable Average Ratings:
```

Comment:
From a design perspective, I believe single words will look best here, so I'd probably change this to "Ratings".